index.d.ts

```diff
@@ -2976,7 +2976,7 @@ declare module sequelize {
     [string | col | fn | literal, string] |
     [Model<any, any> | { model: Model<any, any>, as: string }, string, string] |
     [Model<any, any>, Model<any, any>, string, string];
-  export type Order = OrderItem | OrderItem[];
+  export type Order = string | fn | col | literal | OrderItem[];
```

Comment:
No, like I said, passing a single `OrderItem` like `['name', 'DESC']` is not valid, it would get interpreted as two columns `name` and `DESC` and fail with an column-does-no-exist-error.